### ISSUE TITLE
Can not connect Redis sentinel in private EKS cluster from lambda

### ISSUE THREAD

**Issue by sanjay124** [aws-samples/aws-lambda-layer-kubectl#57]
I'm trying to access a Redis sentinel which is in a private EKS cluster already deployed.
I have assigned a VPC and EKS cluster security group to this lambda function but still no luck.

**Comment by pahud** [comment]
Are you trying to connect redis service from Lambda with a redis client?

**Comment by sanjay124** [comment]
Yes, was using the same code for EC2 but now I migrated Redis on EKS so not able to connect my lambda code to the Redis pod
